src/HtmlGenerator/Utilities/Markup.cs

```diff
@@ -33,11 +33,7 @@ public static string HtmlEscape(string text)
             return text;
         }
 
-        private static string IntersperseLineBreaks(string text)

```

Comment:
The reason this method was written like this is for easier debugging. But your refactoring is fine I guess.